content/widgets/answers.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Answers
+type: widget
+html: |
+  <div class="ais-Answers">
+    <div class="ais-Answers-header">Algolia Answers</div>
+    <ul class="ais-Answers-list">
+      <li class="ais-Answers-item">Author Sam Martin shares photos of ...</li>
+      <li class="ais-Answers-item">Beau Lotto's color games puzzle your vision, but they also spotlight what you can't normally see...</li>
+    </ul>
+  </div>
+alt1: empty
+althtml1: |
+  <div class="ais-Answers ais-Answers--empty">
+    <div class="ais-Answers-header">Algolia Answers</div>
+    <ul class="ais-Answers-list"></ul>
```

Comment:
the empty `ul` should probably not be rendered: https://webaim.org/discussion/mail_thread?thread=1351

Reply:
I'm not sure. I tried but couldn't find anything about it elsewhere. I guess that thread is specific to their usecase where they maintain some nested lists?

Reply:
I can't find much information either, but a list is meant to have items, this way it's going to be announced for screen readers, while there's no content inside

Reply:
@Haroenv I genuinely have no idea which is better for accessibility, but at least we do have empty `ul` across all the IS flavors. If you think it's good to get rid of empty `ul`, then we can have that behavior starting from this PR. What do you think?

Reply:
Good point. For the time being we can do either way since I can't find much information around it either. Empty ul should probably not exist, but likely also doesn't have a huge impact

Reply:
I've just found out a minor usecase:

```css
ul:empty::before {
  content: "Empty!";
}
```

Something like that could be used.